static/docs/commands-reference/add.md

```diff
@@ -144,36 +144,36 @@ Your goal might be to build an algorithm to identify dogs and cats in pictures,
 and this is your training data set:
 
 ```dvc
-    $ tree pics
-    pics
-    ├── train
-    │   ├── cats        <-- a lot of images of cats
-    │   └── dogs        <-- a lot of images of dogs
-    └── validation
-        ├── cats        <-- images of cats
-        └── dogs        <-- images of dogs
+$ tree pics
+pics
+├── train
+│   ├── cats        <-- a lot of images of cats
+│   └── dogs        <-- a lot of images of dogs
+└── validation
+    ├── cats        <-- images of cats
+    └── dogs        <-- images of dogs
 ```
 
 Taking a directory under DVC control as simple as taking a single file:
 
 ```dvc
-    $ dvc add pics
+$ dvc add pics
 
-    Computing md5 for a large directory pics/train/cats. This is only done once.
-    [##############################] 100% pics/train/cats
+Computing md5 for a large directory pics/train/cats. This is only done once.
+[##############################] 100% pics/train/cats
 
-    ...
+...
 
-    Saving 'pics' to cache '.dvc/cache'.
+Saving 'pics' to cache '.dvc/cache'.
 
-    Linking directory 'pics'.
-    [##############################] 100% pics
+Linking directory 'pics'.
+[##############################] 100% pics
 
-    Saving information to 'pics.dvc'.
+Saving information to 'pics.dvc'.
 
-    To track the changes with git run:
+To track the changes with git run:
 
-      	git add pics.dvc
+  	git add pics.dvc
```

Comment:
same question - is it a tab here?

Reply:
Okay, I got the thing with this.
```
$ dvc add pics

Computing md5 for a large directory pics/train/cats. This is only done once.
[##############################] 100% pics/train/cats

...

Saving 'pics' to cache '.dvc/cache'.

Linking directory 'pics'.
[##############################] 100% pics

Saving information to 'pics.dvc'.

To track the changes with git run:

  	git add pics.dvc
```
The output to this command is in this way only.  Firstly I had stripped them, but later, going through other docs, I found the output to this command in the same structure. I reset them back to the original state.
@shcheklein If you want them to be on the extreme left, let me know.

Reply:
@ryokugyu yep, it makes sense. if the command outputs tabs, let's keep. just remove extra spaces.